runtime/runc_jailer.go

```diff
@@ -136,7 +150,7 @@ func (j *runcJailer) BuildJailedRootHandler(cfg *Config, socketPath *string, vmI
 			}
 
 			j.logger.Debug("Overwritting process args of config")
-			if err := j.overwriteConfig(cfg, filepath.Base(m.Cfg.SocketPath), rootPathToConfig); err != nil {
+			if err := j.overwriteConfig(cfg, machineConfig, filepath.Base(m.Cfg.SocketPath), rootPathToConfig); err != nil {
```

Comment:
I'm not positive but I would have thought this would need to be `m.Cfg` as opposed to `machineConfig`. The reason being that `machineConfig` is the one [provided during `buildVMConfiguration` to `BuildJailedRootHandler` and `NewMachine`](https://github.com/firecracker-microvm/firecracker-containerd/blob/5f3e0c74a7c707ac7e0cd0a8f1466abff75d47de/runtime/service.go#L486), but `NewMachine` doesn't modify that `machineConfig` object as its [signature has just `Config`, not `*Config`](https://github.com/firecracker-microvm/firecracker-go-sdk/blob/eec83f3dcb7b4eb4eeb1e2c5f975715a25f79e96/machine.go#L257).

`m.Cfg` should have the fully updated config though if I'm understanding correctly (which I may not be).

Reply:
Whoops, yea, I meant `m.Cfg`. Too many configs lol

Reply:
Fixed!